ibis/expr/rewrites.py

```diff
@@ -91,6 +92,48 @@ def rewrite_dropna(_):
     return ops.Filter(_.parent, tuple(preds))
 
 
+@replace(p.WindowFunction(p.First(x, y)))
```

Comment:
Are these specific to trino?

Reply:
No, other backends that lack a proper `First`/`Last aggregate (but implement it for scalar aggs in Ibis) can reuse this.

Reply:
Can you please move these rules to sqlglot/rewrites.py? 

Later we can do a consolidation pass over all the different rewrites.py